File: java/src/test/java/com/github/agebhar1/ApplicationTests.java
package com.github.agebhar1;

import org.junit.jupiter.api.Test;
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.boot.test.context.SpringBootTest;

import static org.junit.jupiter.api.Assertions.assertEquals;

@SpringBootTest
class ApplicationTests {

    @Test
    void contextLoads() {
    }

    @Test
    void shouldReadProperty(@Autowired MyConfig config) {
        assertEquals("any", config.getImport());
    }

}

File: kotlin/src/test/kotlin/com/github/agebhar1/ApplicationTests.kt
package com.github.agebhar1

import org.junit.jupiter.api.Assertions.assertEquals
import org.junit.jupiter.api.Test
import org.springframework.beans.factory.annotation.Autowired
import org.springframework.boot.test.context.SpringBootTest

@SpringBootTest
class ApplicationTests {

  @Test fun `context loads`() {}

  @Test
  fun `should read property`(@Autowired config: MyConfig) {
    assertEquals("any", config.importValue)
  }
}
